app/views/cannot_upload_another_file.scala.html

```diff
@@ -15,16 +15,23 @@
  *@
 
 @import config.ApplicationConfig
+@this(
+        govuk_wrapper: views.html.govuk_wrapper,
+        govukButton: GovukButton)
 @()(implicit request: Request[_], messages: Messages, appConfig: ApplicationConfig)
-	@ras_main(title = messages("cannot.upload.another.file.page.title")) {
+	@govuk_wrapper(pageTitle = messages("cannot.upload.another.file.page.title")) {
 
-    <div class="row ras-bottom-margin">
-        <h1 id="page-header" class="heading-xlarge">@messages("cannot.upload.another.file.page.header")</h1>
-        <p id="page-clarification">@messages("cannot.upload.another.file.page.clarification")</p>
-    </div>
-
-    <div class="row">
-        <a id="choose-something-else" class="button" href="@routes.ChooseAnOptionController.get" data-journey-click="button - click:You cannot upload another file:Choose something else to do">@messages("choose.something.else")</a>
+    <div id="cannot-upload-another-file" class="govuk-!-margin-bottom-6">
```

Comment:
The class row has been removed from both the div...s. Guessing its not being used / needed?

Reply:
Yep, there was no styling for 'row' so not sure why it was being used as a class.